typescript/src/updater/update_feed.ts

```diff
@@ -0,0 +1,171 @@
+// ## Cli
+// - chain
+// - target-chain (de Pragma.sol - donc ça doit être déployé)
+// - feed_id
+// - private_key
+// - (keystore)
+
+// ## Steps
+// 1. Theoros ? (choix de l'endpoint)
+// 2. Get calldata from Theoros
+// 3. Call `updateDataFeeds` avec la calldata
+// 4. Assertions - check that on the destination chain everything is correctly updated
+//    Show gas consumption
+
+import { Command } from "commander";
+import { ethers } from "ethers";
+import abi from "../../solidity/out/Pragma.sol/Pragma.json";
+
+interface ChainConfig {
+  contractId: string;
+}
+
+function getChainConfig(chainId: string): ChainConfig {
+  const chainConfigs: { [key: string]: ChainConfig } = {
+    "0x1": { contractId: "0x1111111111111111111111111111111111111111" }, // Ethereum Mainnet
+    "0xaa36a7": { contractId: "0x2222222222222222222222222222222222222222" }, // Ethereum Sepolia
+    "0x534e5f4d41494e": {
+      contractId: "0x3333333333333333333333333333333333333333",
+    }, // StarkNet Mainnet
+    "0x534e5f474f45524c49": {
+      contractId: "0x4444444444444444444444444444444444444444",
+    }, // StarkNet Sepolia (Goerli)
+    "0xa4b1": { contractId: "0x5555555555555555555555555555555555555555" }, // Arbitrum
+    "0xa": { contractId: "0x6666666666666666666666666666666666666666" }, // Optimism
+  };
+
+  const config = chainConfigs[chainId];
+  if (!config) {
+    throw new Error(`Configuration non trouvée pour le chainId ${chainId}`);
+  }
+
+  return config;
+}
+
+function parseCommandLineArguments() {
```

Comment:
We should say in the function prototype that we return something - i.e. : 
```typescript
function parseCommandLineArguments(): returnTypeHere {
```

Reply:
done